modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -140,6 +140,14 @@ if (showSource) {
 name = HtmlUtil.escapeJS(name);
 %>
 
+<aui:script require="frontend-js-web/index as frontendJsWeb">
```

Comment:
I consider this as a easy (perhaps hacky) way to obtain the JS we need in the JSP context, in other words, my goal was to prove this technique can be implemented.

We can as well create a JS component in charge of making the object available in the `Liferay.Util` for easy utilization, and use `<liferay-frontend:component>` for that purpose. I'll try that, makes sense.

I see value in keeping this in the same JSP where we need it, as it provides on-demand usage and facilitates code removal when we deprecate this.

Reply:
Oh, and I think that syntax is using two bridge layers :see_no_evil: 

One to convert `frontend-js-web` from unversioned to versioned, another one to convert versioned AMD module to ES module.

So, IINM, this would be better expressed as `frontendJsWeb from frontend-js-web` or something similar... I would need to test it, I'm talking off the top of my head...